node/e2etests/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "azure-iot-e2etests",
+  "version": "1.0.0-preview.11",
```

Comment:
Are we going to publish this package?  I suspect not, and if that's the case then take a look at the other unpublished package.json files in our repo (e.g., node/device/samples) and follow the pattern there (e.g., version: 0.0.1, private: true)